packages/mobile/src/components/delete-chat-drawer/DeleteChatDrawer.tsx

```diff
@@ -94,26 +84,12 @@ export const DeleteChatDrawer = () => {
         <Text style={styles.confirm} allowNewline>
           {messages.description}
         </Text>
-        <Button
-          title={messages.deleteButton}
-          onPress={handleConfirmPress}
-          variant={'destructive'}
-          styles={{
-            root: styles.button,
-            text: styles.blockText
-          }}
-          fullWidth
-        />
-        <Button
-          title={messages.cancel}
-          onPress={closeDrawer}
-          variant={'common'}
-          styles={{
-            root: styles.button,
-            text: styles.blockText
-          }}
-          fullWidth
-        />
+        <Button onPress={handleConfirmPress} variant={'destructive'} fullWidth>
+          {messages.deleteButton}
+        </Button>
+        <Button onPress={closeDrawer} variant={'secondary'} fullWidth>
```

Comment:
nit: don't need the `{'...'}`
```suggestion
        <Button onPress={handleConfirmPress} variant='destructive' fullWidth>
          {messages.deleteButton}
        </Button>
        <Button onPress={closeDrawer} variant='secondary' fullWidth>
```

Reply:
thanks! yes i decided to leave existing issues as is, but can fix this file :)